lib/server.js

```diff
@@ -0,0 +1,19 @@
+import { Server } from 'hapi'
+
+export default (port) => {
+  const server = new Server()
+
+  server.connection({port: port})
+
+  server.start(() => {
+    console.log(`Running at http://localhost:${port}`)
+  })
```

Comment:
Fun. Ah I see now. It's going to pull that from `os.hostname()`. So for me it's:

<img width="574" alt="screen shot 2016-01-04 at 7 40 26 pm" src="https://cloud.githubusercontent.com/assets/590904/12104605/0867a364-b31b-11e5-87cc-058e2dbdc8cc.png">

Which is basically the same as typing `hostname` in the command line. Curiously both of these addresses register for me (http://nate.local:8000 and http://localhost:8000). 

So short of it is that I don't have an answer for you. The next question I want to answer: Is there a drawback to specifying `host: "localhost"` in `server.connection({ ...config })`?

Either way, it looks like I'm in the wrong here.